bindings/python/tests/bindings/test_pre_tokenizers.py

```diff
@@ -195,6 +196,21 @@ def test_can_modify(self):
         assert pretok.individual_digits == True
 
 
+class TestFixedLength:
+    def test_instantiate(self):
+        assert FixedLength() is not None
+        assert isinstance(FixedLength(), PreTokenizer)
+        assert isinstance(FixedLength(), FixedLength)
+        assert isinstance(pickle.loads(pickle.dumps(FixedLength())), FixedLength)
+
+    def test_can_modify(self):
+        pretok = FixedLength(length=5)
+        assert pretok.length == 5
+
+        pretok.length = 10
+        assert pretok.length == 10
+
```

Comment:
we'd also want to make sure that it does it's job as a pretokenizer! so testing with the same string, that it splits in 5 then 10!